Task 01 Anastasia Kravchuk/Task_01.py

```diff
@@ -0,0 +1,142 @@
+"""Task 01."""
+# ====================== IMPORT ======================
+
+import glob
+import re
+import sys
+from time import localtime
+
+# ====================== Line 1 ======================
+
+print('Hello, world!')
+
+# ====================== Line 2 ======================
+
+name = input('What is your name?\n')
+print('Hi, %s.' % name)
+
+# ====================== Line 3 ===========s===========
+
+friends = ['john', 'pat', 'gary', 'michael']
+for i, name in enumerate(friends):
+    print("iteration {iteration} is {name}".format(iteration=i, name=name))
+
+# ====================== Line 4 ======================
+
+parents, babies = (1, 1)
+while babies < 100:
+    print('This generation has {0} babies'.format(babies))
+    parents, babies = (babies, parents + babies)
+
+# ====================== Line 5 ======================
+
+
+def greet(name):
+    """Meeting function."""
+    print('Hello', name)
+
+
+greet('Jack')
+greet('Jill')
+greet('Bob')
+
+# ====================== Line 6 ======================
+
+for test_string in ['555-1212', 'ILL-EGAL']:
+    if re.match(r'^\d{3}-\d{4}$', test_string):
+        print(test_string, 'is a valid US local phone number')
+    else:
+        print(test_string, 'rejected')
+
+# ====================== Line 7 ======================
+
+prices = {'apple': 0.40, 'banana': 0.50}
+my_purchase = {
+    'apple': 1,
+    'banana': 6}
+grocery_bill = sum(prices[fruit] * my_purchase[fruit]
+                   for fruit in my_purchase)
+print('I owe the grocer $%.2f' % grocery_bill)
+
+# ====================== Line 8 ======================
+
+# This program adds up integers in the command line
+
+try:
```

Comment:
Just a tip. 
As this module is pretty big we can put to a docstring information that it could be launched with integer arguments